src/js/actions/superselect.js

```diff
@@ -622,8 +622,12 @@ define(function (require, exports) {
                         }
 
                         var artboardNested = false;
+                        var moveHandled  = false;
```

Comment:
noted.  FYI barkin, I'm going to mess with this because I need to make some historyStore changes to workaround this great photoshop quirk.